### ISSUE TITLE
Vivecraft mutes pc audio then crashes

### ISSUE THREAD

**Issue by ghost** [ferriarnus/VivecraftForge#269]
It was working fine 2 days ago, I start it up now and this happens.
[hs_err_pid77751.log](https://github.com/Vivecraft/VivecraftMod/files/14965353/hs_err_pid77751.log)
any idea why? I can't make sense of this.

**Comment by Techjar** [comment]
The crash is in some SteamVR native library code, so it's out of our control. Could be a bugged SteamVR update, or maybe something weird with your PC/OS, no idea honestly.

**Comment by fayer3** [comment]
you are running steamvr first?

**Comment by ghost** [comment]
> you are running steamvr first?

Yes, steamvr was running, but it might be a bit buggy. I tried other games like Serious Sam TFE VR and H3VR. Those games didn't launch until I restarted Steamvr.
